### PR TITLE
Removed Recommendation to Use Anaconda3.

### PR DESCRIPTION
**Pull request by deeleeramone**
Anaconda3 will source dependencies from different places than Miniconda and can create conflicts with our dependency tree.

### CODE REVIEW

**Comment by deeleeramone** [comment]
Would it be reasonable to recommend GWSL? https://opticos.github.io/gwsl/

File: website/content/terminal/quickstart/installation.md [hunk 1]
```diff
@@ -245,7 +245,7 @@ We can use IPC socket to connect Desktop.
 Add this setting to your `.env` file.
 
 ```bash
-OPENBB_BACKEND=Qt5Agg
```
Comment (piiq) [inline website/content/terminal/quickstart/installation.md:248]: Tkinter (Tk) is the default backend on most linux distros. Instructions to use Qt as a matplotlib backend was contributed by a user of some rare linux distro. I suggest this setting goes into the troubleshooting section and not be present in the general documentation

Comment (deeleeramone) [reply website/content/terminal/quickstart/installation.md:248]: Removed from this section.

**Review by JerBouma — CHANGES_REQUESTED** [review]

File: website/content/terminal/quickstart/installation.md [hunk 2]
```diff
@@ -264,36 +256,33 @@ Then run the previous docker command.
...
+These steps are common in all operating systems (Windows with or without WSL, MacOS or Linux).
 
-**NOTE:** We recommend using `conda` and `poetry` because it just works. You can use other python
-distributions and use raw `pip` instead of `poetry` but you will very likely bump into installation
-issues.
+To install via Python, a virtual environment must be created. This isolates the packages installed from the rest of the system. It is our recommendation that you utilize a Conda environment because there are optional features, such as forecast, that utilize libraries specifically sourced from conda-forge. Due to this, if you do not use a Conda environment, you will not be able to use some of these features. As such, the installation steps will be written under the assumption that you are using Conda.
```
[the remark this answers is not in the document]
Comment (JerBouma) [reply website/content/terminal/quickstart/installation.md:267]: I would phrase it as `To install via Python, we recommend creating an additional virtual environment.`

[the remark this answers is not in the document]
Comment (piiq) [reply website/content/terminal/quickstart/installation.md:267]: I agree with your wording without `additional`

[the remark this answers is not in the document]
Comment (JerBouma) [reply website/content/terminal/quickstart/installation.md:267]: All good with me.

File: website/content/terminal/quickstart/installation.md [hunk 3]
```diff
@@ -264,36 +256,33 @@ Then run the previous docker command.
...
 
 #### 1. [Install Miniconda](https://docs.conda.io/en/latest/miniconda.html)
```
[the remark this answers is not in the document]
Comment (deeleeramone) [reply website/content/terminal/quickstart/installation.md:269]: Done

**Review by JerBouma — APPROVED** [review]

**Comment by jmaslek** [comment]
Pinging @deeleeramone again :D

**Comment by deeleeramone** [comment]
> Pinging @deeleeramone again :D

I am also unclear about these conflicts. Most of changes were capitalizing words like docker and python, but the conflicts are showing the order of things has changed.  There is also another open PR editing this file, so I think the best thing might be close this PR, let the other edits in and come back to this is necessary.

**Pull request closed without merging**